app/routes.js

```diff
@@ -1,9 +1,10 @@
-// const controller = require('./controllers/controller');
 const { healthCheck } = require('./controllers/healthCheck');
+const { signUp } = require('./controllers/users');
+const signUpSchema = require('./schemas/sign_up');
+
+const validationSchema = require('./middlewares/validation_schema');
 
 exports.init = app => {
   app.get('/health', healthCheck);
-  // app.get('/endpoint/get/path', [], controller.methodGET);
-  // app.put('/endpoint/put/path', [], controller.methodPUT);
-  // app.post('/endpoint/post/path', [], controller.methodPOST);
+  app.post('/users', signUpSchema(), validationSchema, signUp);
```

Comment:
```suggestion
  app.post('/users', [signUpSchema(), validationSchema], signUp);
```

Reply:
I know ir is the same but it's easier to read if you have the middlewares array explicitly